WcaOnRails/app/views/polls/results.html.erb

```diff
@@ -0,0 +1,42 @@
+<% provide(:title, 'Results for poll #' + @poll.id.to_s) %>
+
+<div class="container">
+
+  <h3><%= @poll.question %></h3>
+
+  <% @vote = @poll.votes.find_by user_id: current_user %>
+
+  <% if @poll.poll_is_over? %>
+    <table class="table" id="poll-results">
+      <% @poll.poll_options.each do |option| %>
+        <tr>
+          <td>
+            <% if @vote && @vote.poll_option_id == option.id %>
+              <span class="glyphicon glyphicon-ok-sign" aria-hidden="true" data-toggle="tooltip" data-placement="top" data-container="body" title="You have voted in this option"></span>
+            <% end %>
+          </td>
+          <td><%= option.description %></td>
+          <td><%= option.votes.count %></td>
+          <td><%= (option.votes.count.to_f/@poll.votes.count*100).round(2) %>%</td>
```

Comment:
Yes, I can.

But I'll have to fix the voting counting because we changed the models.